test/python/test_tensor.py

```diff
@@ -0,0 +1,159 @@
+import sys, os
+import numpy as np
+
```

Comment:
For unitest, we can use this one https://docs.python.org/2.7/library/unittest.html.
It is similar to gtest, we use assertEqual to check the execution result the and expected result.